edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/BookingData.java

```diff
@@ -0,0 +1,299 @@
+  name = "graph.booking-data",
+  attributeNodes = {
+    @NamedAttributeNode("vessel"),
+    @NamedAttributeNode("placeOfIssue"),
+    @NamedAttributeNode("invoicePayableAt"),
+    @NamedAttributeNode("references"),
+    @NamedAttributeNode(value = "documentParties", subgraph = "graph.documentParties"),
+    @NamedAttributeNode("shipmentLocations"),
+    @NamedAttributeNode("shipmentCutOffTimes"),
+    @NamedAttributeNode("carrierClauses"),
+    @NamedAttributeNode("confirmedEquipments"),
+    @NamedAttributeNode("charges"),
+    @NamedAttributeNode("carrier"),
+  },
+  subgraphs = {
+    @NamedSubgraph(
+      name = "graph.documentParties",
+      attributeNodes = {
+        @NamedAttributeNode("displayedAddress"),
+        @NamedAttributeNode(value = "party", subgraph = "graph.party")
+      }),
+    @NamedSubgraph(
+      name = "graph.party",
+      attributeNodes = {@NamedAttributeNode("partyContactDetails")}),
+    @NamedSubgraph(
+      name = "subgraph.shipmentLocations",
+      attributeNodes = {@NamedAttributeNode("location")}),
+    @NamedSubgraph(
+      name = "subgraph.shipmentTransports",
+      attributeNodes = {
+        @NamedAttributeNode("loadLocation"),
+        @NamedAttributeNode("dischargeLocation"),
+      }),
+  })
+@Slf4j
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PRIVATE)
+@Entity
+@Table(name = "booking_data")
+@BookingDataValidation(groups = AsyncShipperProvidedDataValidation.class)
+public class BookingData {
+
+  @Id
+  @Column(name = "id", nullable = false)
+  @GeneratedValue
+  private UUID id;
+
+  @Column(name = "receipt_type_at_origin")
+  @Enumerated(EnumType.STRING)
+  private ReceiptDeliveryType receiptTypeAtOrigin;
+
+  @Column(name = "delivery_type_at_destination")
+  @Enumerated(EnumType.STRING)
+  private ReceiptDeliveryType deliveryTypeAtDestination;
+
+  @Column(name = "cargo_movement_type_at_origin")
+  @Enumerated(EnumType.STRING)
+  private CargoMovementType cargoMovementTypeAtOrigin;
+
+  @Column(name = "cargo_movement_type_at_destination")
+  @Enumerated(EnumType.STRING)
+  private CargoMovementType cargoMovementTypeAtDestination;
+
+  @Column(name = "service_contract_reference", length = 30)
+  private String serviceContractReference;
+
+  @Column(name = "payment_term_code")
+  @Enumerated(EnumType.STRING)
+  private PaymentTerm paymentTermCode;
+
+  @Column(name = "is_partial_load_allowed")
+  private Boolean isPartialLoadAllowed;
+
+  @Column(name = "is_export_declaration_required")
+  private Boolean isExportDeclarationRequired;
+
+  @Column(name = "export_declaration_reference", length = 35)
+  private String exportDeclarationReference;
+
+  @Column(name = "is_import_license_required")
+  private Boolean isImportLicenseRequired;
+
+  @Column(name = "import_license_reference", length = 35)
+  private String importLicenseReference;
+
+  @Column(name = "is_ams_aci_filing_required")
+  private Boolean isAMSACIFilingRequired;
```

Comment:
I am quite sure we don't need this any more. It has been replaced by advancedManifestFilings - is this a TODO?

Reply:
Matching existing behaviour 1:1, so it would be a TODO. I expect to clean it up during the "auto-generated TO" step.